lib/icinga/scheduleddowntime.cpp

```diff
@@ -171,7 +171,7 @@ std::pair<double, double> ScheduledDowntime::FindRunningSegment(double minEnd)
 
 		if (!bestSegment || end > bestEnd) {
 			Log(LogDebug, "ScheduledDowntime") << "(best match yet)";
-			bestSegment = segment;
+			bestSegment.swap(segment);
```

Comment:
Should also just move (two more uses of swap in the PR that could just be moves).